package.json

```diff
@@ -63,7 +64,7 @@
   },
   "scripts": {
     "build": "react-scripts build",
-    "build-local-dev-config": "cat .env.common-local .env.dev > .env.local",
+    "build-local-dev-config": "cat .env.dev .env.common-local > .env.local",
```

Comment:
Why did these get swapped? Do we need to swap the others?

Reply:
Mmmh, probably. I think the `.env.common-local` file should be appended so that it overrides other variables. For example, I want the `test` sentry project locally, rather the the sentry project for the hosted dev.